doc/rainbow-delimiters.txt

```diff
@@ -157,7 +162,7 @@ available as part of a Lua module.
         blacklist = {'c', 'cpp'},
     }
 <
-The keys are exactly the same as for |g:rainbow_parentheses|.  If fact, this
+The keys are exactly the same as for |vim.g.rainbow_delimiters|.  In fact, this
```

Comment:
Thanks for catching this mistake, but please keep global Vim variables as `g:` variables because that is what the manual does.

```suggestion
The keys are exactly the same as for |g:rainbow_delimiters|.  In fact, this
```

Reply:
Ok, I will change it back. Just as a note: The reason I changed it here was to distinguish between the vim code and Lua code above it, where the Lua code uses `vim.g.rainbow_delimiters`, but is should be pretty obvious what is meant here anyways.